src/main/resources/application.yml

```diff
@@ -40,6 +40,7 @@ spring:
     store-type: none
     jdbc:
       initialize-schema: never
+  cloud.mvc.token-relay.enabled: false
```

Comment:
What does it affect?

Reply:
It uses a session so it forces one to be created. It didn't seem to be used anywhere, at least downstream OAuth2 calls still work fine even when this is disabled.